client/src/components/ReceiveRole.svelte

```diff
@@ -0,0 +1,124 @@
+<script lang="ts">
+  import {
+    GAME_STATE_DURATIONS,
+    type Lobby,
+    type Player,
+    type PrivatePlayer,
+  } from "$lib/firebase/firestore-types/lobby";
+  import { onMount } from "svelte";
+  import { authStore as user } from "$stores/auth";
+  import { verifyExpiration } from "$lib/firebase/firebase-functions";
+
+  export let lobby: Lobby;
+  export let lobbyCode: string;
+  export let privatePlayer: PrivatePlayer;
+
+  let countdown = GAME_STATE_DURATIONS.VOTE;
+  let timer: ReturnType<typeof setInterval>;
+  let votedOffUser: Player | undefined;
```

Comment:
not used variable

Reply:
I noticed that I forgot to delete the voted off stuff too